### DISCUSSION TITLE
Delete Library Option and Content Files

### DISCUSSION THREAD

**Discussion by kulak** [stumpapp/stump#523] [General]
I see this option and I am guessing that this will delete Library AND book files.  Or will it only delete Library and keep files untouched?

![image](https://github.com/user-attachments/assets/f3327549-0a5e-4268-90df-4f852a7855dd)

This option shall clearly describe the impact. 

I don't think it is a good idea to remove files represented by library, because what if user decides to just re-create library with a different scan option or make some other configuration changes to the library.  It would be a very nasty surprise that book files are gone.

**Comment by aaronleopold** [comment]
This does not touch the underlying files, see https://www.stumpapp.dev/guides/basics/libraries#delete-a-library:

> Deleting a library will remove all of the series, books, and other entities associated with the books (e.g., read progress, reading lists, etc) contained within it. Your actual files will not be deleted, but they will no longer exist in Stump.

I can rephrase the messaging to be more concise in the UI, directly.

**Reply by kulak** [reply]
I kind of got scared and mounted data directory as read only to prevent accidents.
